ui/src/app/machines/views/MachineDetails/MachineSummary/OverviewCard/StatusCard/StatusCard.tsx

```diff
@@ -101,7 +102,12 @@ const StatusCard = ({ machine }: Props): JSX.Element => {
             <hr />
             <p className="u-text--muted">
               Periodic hardware sync enabled{" "}
-              <Tooltip
+              <TooltipButton
+                buttonProps={{
+                  "aria-label": "more about periodic hardware sync",
+                }}
```

Comment:
Adding an aria-label will be a _very_ common use case (as we're mostly relying on just icons for these buttons so we need labels in most cases).

I can't imagine why you'd ever want/need to attach aria-label an element other than `button`. I'd suggest to simplify and make it a top-level prop.

```suggestion
                aria-label="more about periodic hardware sync"
```

Reply:
Sure thing, sounds good to me :+1: